ddtrace/span.py

```diff
@@ -49,13 +49,13 @@ def __init__(
         span_id=None,
         parent_id=None,
         start=None,
-        ctx=None,
+        context=None,
```

Comment:
It won't work without it right? I'd say yes then (like `tracer`).
Won't affect user's instrumented code.